manifests/files.pp

```
@@ -51,14 +52,15 @@
     subscribe   => File['/etc/aliases'],
   }
 
-  # Config files
+  # Config files - mastercf_source wins over mastercf_content
```

I think I'd rather raise an error if both are provided.